community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/DynamicRecord.java

```diff
@@ -22,17 +22,18 @@
 import org.neo4j.kernel.impl.store.PropertyStore;
 import org.neo4j.kernel.impl.store.PropertyType;
 
-
-public class DynamicRecord extends Abstract64BitRecord
+public class DynamicRecord extends AbstractBaseRecord
 {
+    private static final byte[] NO_DATA = new byte[0];
     private static final int MAX_BYTES_IN_TO_STRING = 8, MAX_CHARS_IN_TO_STRING = 16;
 
-    private byte[] data = null;
+    private byte[] data;
     private int length;
-    private long nextBlock = Record.NO_NEXT_BLOCK.intValue();
+    private long nextBlock;
     private int type;
-    private boolean startRecord = true;
+    private boolean startRecord;
 
+    @Deprecated
```

Comment:
can we remove those 2 deprecated methods or add javadoc what to do if you are still using those?